test/e2e/specs/fakerp/admin.go

```diff
@@ -1,23 +1,33 @@
-//+build e2e
-
-package e2e
+package fakerp
 
 import (
 	"strings"
 	"time"
 
-	"github.com/ghodss/yaml"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	kerrors "k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/ghodss/yaml"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/openshift/openshift-azure/pkg/jsonpath"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
 )
 
 var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader]", func() {
-	defer GinkgoRecover()
+	var (
+		cli *openshift.Client
+	)
+
+	BeforeEach(func() {
+		var err error
+		cli, err = openshift.NewSystemAdminClient()
```

Comment:
This is wrong, we should be using the azure cluster reader client and not admin.

Reply:
we need an extra `openshift.NewAzureClusterReaderClient()`